dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/publish/servers/ReactiveStreamsTestServer/server.xml

```diff
@@ -2,18 +2,24 @@
 
 	<include location="../fatTestPorts.xml" />
 
-        <variable name="testVar" value="testValue"/>
+	<variable name="testVar" value="testValue" />
 
 	<featureManager>
 		<feature>mpReactiveStreams-1.0</feature>
-    	<feature>osgiconsole-1.0</feature>
+	 	<feature>mpConfig-1.3</feature>
+		<feature>osgiconsole-1.0</feature>
 		<feature>cdi-2.0</feature>
-	 	<feature>servlet-4.0</feature> 
+		<feature>servlet-4.0</feature>
 		<feature>componentTest-1.0</feature>
 	</featureManager>
 
-   <application id="mpReactiveStreamsFAT" name="ReactiveStreamsTest" type="war" location="ReactiveStreamsTest.war">
-   </application>
+	<application id="mpReactiveStreamsFAT"
+		name="ReactiveStreamsTest" type="war"
+		location="ReactiveStreamsTest.war">
+		<classloader apiTypeVisibility="+third-party" />
+	</application>
+	<javaPermission className="java.lang.RuntimePermission"
```

Comment:
which bit of code needed this permission? if it was in the runtime code then a doPriv might be better

Reply:
I believe it was something in Arquillian (so won;t change the code) - it only turns up in the Java 2 security build fat tests.